dev-tex/rubber/rubber-1.5.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_6 )
+
+inherit distutils-r1
+
+DESCRIPTION="A LaTeX wrapper for automatically building documents"
+HOMEPAGE="https://launchpad.net/rubber/"
+SRC_URI="https://launchpad.net/rubber/trunk/${PV}/+download/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+RDEPEND="virtual/latex-base"
+DEPEND="${RDEPEND}
+	virtual/texi2dvi"
+
+python_install() {
+	distutils-r1_python_install \
```

Comment:
To make it more elegant without breaking lines by ```\```, you could try to add the into an array.

Reply:
If what I assume you to suggest is right, then I would prefer the current variant.

Reply:
That's fine, since it's only a suggestion. I you like it, keep it.